tracing-attributes/src/lib.rs

```diff
@@ -176,6 +176,31 @@ mod expand;
 /// }
 /// ```
 ///
+/// To record a returned value from a function, you can use `ret`:
+///
+/// ```
+/// # use tracing_attributes::instrument;
+/// #[instrument(ret)]
+/// fn my_function() -> i32 {
+///     42
+/// }
+/// ```
+///
```

Comment:
also, maybe worth adding something like
```suggestion
/// The return value event will have the same level as the span generated by `#[instrument]`.
/// By default, this will be `TRACE`, but if the level is overridden, the event will be at the same
/// level.
///
```

Reply:
That's obviously worth doing :+1: